### PR TITLE
Adding only the necessary dependencies for the store-state

### PR DESCRIPTION
**Pull request by CarlaCosenza**
Now the CMS will send us the list of dependencies, so we can add only the necessary ones to the manifest.

### CODE REVIEW

File: node/util/manifest.ts [hunk 1]
```diff
@@ -155,3 +114,31 @@ export async function makeDefaultManifest(
...
+function updateDependencies(
+  dependencies: { [key: string]: string },
+  uploadFile: UploadFile
+) {
+  // tslint:disable-next-line:forin
+  for (const element of uploadFile.dependencies) {
+    const [depAndVar] = element.split(':')
+    const [dep] = depAndVar.split('@')
+    // eslint-disable-next-line prefer-destructuring
+    const ver = depAndVar.split('@')[1]
```
Comment (CarlaCosenza) [inline node/util/manifest.ts:138]: I did not know how to make this comply to the prettier

File: node/util/manifest.ts [hunk 2]
```diff
@@ -155,3 +114,31 @@ export async function makeDefaultManifest(
...
+  manifest.dependencies = updateDependencies(manifest.dependencies, uploadFile)
+
+  return manifest
+}
+
+function updateDependencies(
+  dependencies: { [key: string]: string },
+  uploadFile: UploadFile
+) {
+  // tslint:disable-next-line:forin
```
Comment (mateuspontes) [inline node/util/manifest.ts:133]: we don't need this message here because it's a for of
```suggestion
```

File: node/util/manifest.ts (hunk 1 above)
Comment (mateuspontes) [inline node/util/manifest.ts:138]: I think you can destruct the array in two variables:

```suggestion
    const [dependency, version] = depAndVar.split('@')
```

Note: I think we need to check that version exists next line.

File: node/util/manifest.ts [hunk 3]
```diff
@@ -35,48 +37,6 @@ export function makeEmptyManifest() {
...
-    'vtex.product-list': '0.x',
-    'vtex.add-to-cart-button': '0.x',
-    'vtex.product-bookmark-interfaces': '1.x',
-    'vtex.slider-layout': '0.x',
-    'vtex.store-image': '0.x',
-    'vtex.store-icons': '0.x',
-    'vtex.modal-layout': '0.x',
-    'vtex.store-link': '0.x',
-    'vtex.product-gifts': '0.x',
-    'vtex.product-price': '1.x',
```
Comment (mateuspontes) [inline node/util/manifest.ts:79]: nice 💯

**Pull request merged**